clang/lib/DPCT/APINamesMath.inc

```diff
@@ -600,3 +600,54 @@ ENTRY_RENAMED("mul24", MapNames::getClNamespace(false, true) + "mul24")
 
 // impl with new rewriter
 ENTRY_REWRITE("rsqrt")
+
+// vector types
+ENTRY_RENAMED("make_char1", "char")
```

Comment:
I think these entries would be clearer if they were grouped in this order:

char/unsigned char [1-4]
short/unsigned short [1-4]
int/unsigned int [1-4]
long/unsigned long [1-4]
long long/unsigned long long [1-4]
float [1-4]
double [1-4]